README.md

```diff
@@ -25,7 +25,9 @@ plugins:
 
 ## Configuration
```

Comment:
Can you extend the configuration example with:
* Minimum predefinedMetric
* Full with predefinedMetric
* Minimum with customMetric
* Full with customMetric